doc/developer/contributorAgreements/README.md

```diff
@@ -0,0 +1,18 @@
+=============================
+Chapel Contributor Agreements
+=============================
+
+This directory contains Chapel's Apache 2.0 contributor license
+agreements (CLAs).  We require these to be filled out before
+contributing to the repository in order to help us maintain Chapel's
+license terms.
+
+There are two versions:
+
+* chapel-ccla.pdf : For corporations or other organizations in which
```

Comment:
`[chapel-ccla.pdf](chapel-ccla.pdf)` will relative link to the pdf if that's what you wanted.